e2e/cypress/integration/messaging/draft_with_only_2_byte_characters_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let testChannel;
+    let receiver;
+    let lastPostId;
+
+    before(() => {
+        // # Login as test user
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            receiver = user;
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiLogin(receiver);
+
+            // # Visit a test channel and post a message
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+            cy.postMessage('휴');
+
+            // # Assign lastPostId variable to the id of the last post
+            cy.getLastPostId().then((postId) => {
+                lastPostId = postId;
+            });
+        });
+    });
+
+    it('M23362 - Draft with only 2 byte characters shouldn\'t remain after posting', () => {
+        testChannel.name = testChannel.name.replace(/\s+/g, '-').toLowerCase();
+
+        // * Check that the Korean message got posted in the channel
+        cy.get(`#post_${lastPostId}`).should('exist');
```

Comment:
Seems a bit short in checking whether the message got posted. I suggests to add:
```javascript
cy.get(`#postMessageText_${postId}`).should('contain', '휴');
```